sm_project/cl_app/admin.py

```diff
@@ -43,10 +43,10 @@ class ChecklistItemAdmin(admin.ModelAdmin):
     """Organises how the information about each checklist item is displayed on the admin site."""
     fieldsets = (
         (None, {
-        'fields': ('item_title', 'item_checklist', 'item_status')
+        'fields': ('item_title', 'item_checklist', 'item_status', 'estimate')
```

Comment:
might be clearer to explicitly name what it's an estimate for, e.g. `time_estimate` etc. just a personal preference though